src/v/storage/kvstore.h

```diff
@@ -96,6 +96,7 @@ class kvstore {
         consensus = 1,
         storage = 2,
         controller = 3,
+        coproc = 4,
```

Comment:
these id reservations are for the existing kvstore metadata instance. since you have a different kvstore instance you can start at 1.